app/views/taxons/_organisations.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if presented_organisations.show_organisations? %>
-  <div id="organisations" class="taxon-page__section-group taxon-page__section-group--organisation" data-module="toggle">
+  <div id="organisations" class="taxon-page__section-group taxon-page__section-group--organisation" data-ecommerce-start-index="1" data-list-title="Taxonomy browse" data-search-query="bloop" data-module="toggle">
```

Comment:
If we want to not have organisation tracking, I should remove this set of attributes